this patch disables ez_setup (dpkg handles dependencies a lot better)
Index: simplejson-2.1.0/setup.py
===================================================================
--- simplejson-2.1.0.orig/setup.py
+++ simplejson-2.1.0/setup.py
@@ -1,11 +1,11 @@
 #!/usr/bin/env python
 
 import sys
-try:
-    from ez_setup import use_setuptools
-    use_setuptools(version='0.6c11')
-except ImportError:
-    pass
+#try:
+#    from ez_setup import use_setuptools
+#    use_setuptools(version='0.6c11')
+#except ImportError:
+#    pass
 
 from setuptools import setup, find_packages, Extension, Feature
 from distutils.command.build_ext import build_ext
